packages/importapi-sdk/test/helpers/ctp-api-helper.ts

```diff
@@ -1,46 +1,33 @@
-import {
-  createClient,
-  createAuthForClientCredentialsFlow,
-  createHttpClient,
-} from '@commercetools/sdk-client-v2'
+import { ClientBuilder } from '@commercetools/ts-client'
+import { createApiBuilderFromCtpClient } from '../../src'
 import { requireEnvVar } from './test-utils'
-import {
-  ApiRoot,
-  executeRequest,
-  createExecutorFromMiddlewares,
-} from './../../src'
 
 const projectKey = requireEnvVar('CTP_PROJECT_KEY')
 const clientId = requireEnvVar('CTP_CLIENT_ID')
 const clientSecret = requireEnvVar('CTP_CLIENT_SECRET')
 const authURL = requireEnvVar('CTP_AUTH_URL')
-const ctp_host = requireEnvVar('CTP_IMPORT_URL')
+const ctp_host = requireEnvVar('CTP_HISTORY_URL')
```

Comment:
could it be that there was a confusion here?
Here we are in the import-api package and this variable should be CTP_IMPORT_URL